src/libcore/convert.rs

```diff
@@ -48,25 +48,6 @@
 
 #![stable(feature = "rust1", since = "1.0.0")]
 
-use fmt;
-
-/// A type used as the error type for implementations of fallible conversion
-/// traits in cases where conversions cannot actually fail.
-///
-/// Because `Infallible` has no variants, a value of this type can never exist.
-/// It is used only to satisfy trait signatures that expect an error type, and
-/// signals to both the compiler and the user that the error case is impossible.
-#[unstable(feature = "try_from", issue = "33417")]
-#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
-pub enum Infallible {}
```

Comment:
Looking through that list of derives, are we missing `Copy` and `Clone` for `!`?  They're not in the rustdoc, at least: https://doc.rust-lang.org/nightly/std/primitive.never.html#implementations

(Not a blocker for this PR, of course.)

Reply:
They're implemented. No idea why they're not showing up in rustdoc.

Reply:
Ah, looks like this is a nearing-three-year-old bug with primitives: https://github.com/rust-lang/rust/issues/25893